### PR TITLE
Resize sounding cores

### PR DESCRIPTION
**Pull request by Capkirk123**
Resize the aerobee and WAC science cores to more accurate sizes.

### CODE REVIEW

**Comment by DRVeyl** [comment]
Can you look at extracting the VSR/Restock-specific elements?  (Can we actually observe the difference between a rescale of .1205 vs .12?)  Similarly, can you normalize the model scale to y=1 and then rely on rescaleFactor so that we can more easily rescale without needing to manually touch attachment nodes?  If we are declaring a part from scratch then we need the nodes right, but if we are copying any part that stock or some mod has made, it's on them to put the nodes where they belong.

**Comment by Capkirk123** [comment]
I just wanna resize the parts man (yeah, I can use rescale factor instead)

File: GameData/RealismOverhaul/Parts/Avionics/Sounding0-3m.cfg [hunk 1]
```diff
@@ -139,34 +118,28 @@
...
-	%node_stack_bottom = 0, -0.063158, 0, 0.0, -1.0, 0.0, 0
+	%rescaleFactor = 0.48336
 
-	@DRAG_CUBE
-	{
-		@rescaleX = 0.15
-		@rescaleY = 0.252632
-		@rescaleZ = 0.15
-	}
+	%rescaleCube = 0.48336
```
Comment (DRVeyl) [inline GameData/RealismOverhaul/Parts/Avionics/Sounding0-3m.cfg:142]: `rescaleCube` doesn't do what you hoped.  It is just a flag for if the cube should get rescaled by the `rescaleX`/`rescaleY`/`rescaleZ` values inside the DRAG_CUBE node, it does not treat as if those individual values = the `rescaleCube` value.

Please either just delete the DRAG_CUBE node and allow the PartLoader to auto-generate one, OR to preserve the hollow-ness of the part, edit such that 
```
rescaleX = #$../MODEL/scale[0]$
rescaleY = #$../MODEL/scale[1]$
rescaleZ = #$../MODEL/scale[2]$
@rescaleX *= #$../rescaleFactor$
@rescaleY *= #$../rescaleFactor$
@rescaleZ *= #$../rescaleFactor$
```
I _think_ that will do the correct thing.

**Comment by EagleEyeLogic** [comment]
Not sure what you are doing and how you have tested the changes...

For me with a simple RO installation the cores have 0.954m or 1m. Changing RO from master to the latest release solve this.

screenshot0

**Pull request merged**